storage/storagedriver/select-storage-driver.md

```diff
@@ -55,7 +55,7 @@ Docker supports the following storage drivers:
    driver is poor, and is not generally recommended for production use.
 
 Docker's source code defines the selection order. You can see the order at
-[the source code for Docker Engine - Community {{ site.docker_ce_version }}](https://github.com/docker/docker-ce/blob/{{ site.docker_ce_version }}/components/engine/daemon/graphdriver/driver_linux.go#L50)
+[the source code for Docker Engine - Community (https://github.com/docker/docker-ce/blob/master/components/engine/daemon/graphdriver/driver_linux.go#L50)
```

Comment:
Yes, I think it'll be better to point to the version of the current release; we can remove the ` - Community` as well (it was there to differentiate between "enterprise" and "community"). Lines moved a bit as well, so changed that below to `#L52-L53`;

```suggestion
[the source code for Docker Engine {{ site.docker_ce_version }}](https://github.com/docker/docker-ce/blob/{{ site.docker_ce_version }}/components/engine/daemon/graphdriver/driver_linux.go#L52-L53)
```